app/components/modals/simple-modal.hbs

```diff
@@ -13,7 +13,7 @@
       {{t-var (or @actionName 'OK')}}
     </button>
   {{/if}}
-  <button type="button" class="ui button" {{action 'close'}}>
+    <button type="button" class="ui button" {{action 'close'}} {{action @cancel}}>
```

Comment:
@iamareebjamal confuse here, how to use OR condition between actions please suggest